src/index.html

```diff
@@ -65,6 +65,10 @@ <h1>2048</h1>
         </p>
       </div>
     </div>
-    <script src="scripts/main.js"></script>
+    <script
+      src="scripts/main.js"
+      type="module"
+      defer
```

Comment:
In HTML, when you're using a JavaScript module with the type="module" attribute in a <script> tag, the script is automatically deferred by default, so explicitly adding the defer attribute is not necessary.